New Bank/NewBank/newbank/server/NewBank.java

```diff
@@ -13,23 +14,31 @@ private NewBank() {
 	}
 
 	private void addTestData() {
+		//NewBank bank = new NewBank();
+
 		Customer bhagy = new Customer();
 		bhagy.addAccount(new Account("Main", 1000.0));
 		customers.put("Bhagy", bhagy);
-		
+		bhagy.setBankName("bank3");
```

Comment:
Is there a need to set a new property can CustomerID be used?

Reply:
We cannot use the Customer ID because we are concerned with the bank that we are transferring to.